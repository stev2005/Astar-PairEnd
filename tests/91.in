AGTGGTACGCTGCGCGACGTCACGTATTTCAATTG
15
GTTCAGTCAATACCTGC
AACTAACAAAATGGGCT
TGTT
GTT
TCATGAACACGCCTCAC
GGTACCTAACTTAGCAGT
GACTT
CGTCTTACGAGTAG
CC
GAAGGTTATCA
TCATGTGGAATGTTAT
GGGGGCACAAACATCCTAATGG
GCGCAGACAG
GAAGCCTGGT
TCATCT
