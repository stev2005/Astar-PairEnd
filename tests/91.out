8
10
1
1
7
8
1
5
1
4
7
10
2
4
2
